growthstreet_zoo/zoo.py

```diff
@@ -121,12 +98,11 @@ def __repr__(self):
         return '<%s: %s (%s)>' % (self.__class__.__name__, self, self.species)
 
     def is_eaten(self, animal):
-        """Return a Boolean indicating whether an instance of an animal is eaten."""
+        """Print a string indicating whether an instance of an animal is eaten by."""
```

Comment:
As per our discussion, I don't think this is the best name for this method, as it implies we are just getting a state, rather than performing an action.